src/lib.rs

```diff
@@ -833,6 +869,7 @@ mod env_test {
             "/whatever${VAR}${VAR}" => "/whatevervaluevalue",
             "${VAR} ${VAR}" => "value value",
             "${VAR}$VAR" => "valuevalue",
+            "/answer/${UNKNOWN:-42}" => "/answer/42",
```

Comment:
I'd appreciate more tests, with known variables too, and with various positions of `:-` within the string.

Reply:
I've added more tests to check edge cases